docs/src/components/HeaderLink.tsx

```diff
@@ -39,15 +44,11 @@ const FancyLink = ({ href }: Props) => {
     }
   };
   return (
-    <ButtonLink
-      external
-      href={href}
-      iconLeft={getUsedIcon()}
-      iconRight={<NewWindow ariaLabel="Opens in new window" />}
```

Comment:
Without this, the accessibility of the link decreases -- there's no warning it will open a new window (see [WCAG recommendations](https://www.w3.org/TR/WCAG20-TECHS/G201.html)). That was the point of the icon, not just that it's an external site. I made a note in the design review doc about it too.

Reply:
I agree, so I brought back the icon.